### ISSUE TITLE
[BUG] `LSTMFCNClassifier` `load_model` error: object has no attribute 'model_'

### ISSUE THREAD

**Issue by liudexiang3218** [sktime/sktime#5324]
**Describe the bug**
error : 'LSTMFCNClassifier' object has no attribute 'model_'

**To Reproduce**

```python
from sktime.classification.deep_learning.lstmfcn import LSTMFCNClassifier
from sktime.utils import mlflow_sktime  
from sktime.datasets import load_italy_power_demand
X_train, y_train = load_italy_power_demand(split="train", return_type="numpy3D")
X_new, _ = load_italy_power_demand(split="test", return_type="numpy3D")
classifier=LSTMFCNClassifier(verbose=2,n_epochs=5)
classifier.fit(X_train,y_train)
mlflow_sktime.save_model(sktime_model=classifier,path='classifier')
classifier=mlflow_sktime.load_model(model_uri='classifier') 
classifier.predict(X_new)
```

**Versions**
sktime 0.22.0

**Comment by achieveordie** [comment]
@liudexiang3218 Thanks for bringing it to our notice. I'll get this fixed soon. However, if you're not restricted to only use `mlflow`, I recommend you to use the sktime-native saving method:

``` python
from sktime.classification.deep_learning.lstmfcn import LSTMFCNClassifier
from sktime.datasets import load_italy_power_demand
from sktime.base import load

X_train, y_train = load_italy_power_demand(split="train", return_type="numpy3D")
X_new, _ = load_italy_power_demand(split="test", return_type="numpy3D")

classifier=LSTMFCNClassifier(verbose=2,n_epochs=5)
classifier.fit(X_train,y_train)

classifier.save("estimator")
new_classifier = load("estimator")

new_classifier.predict(X_new)
```

**Comment by achieveordie** [comment]
This should be resolved by #5526.
